### PR TITLE
Marko/avail rust light

### PR DESCRIPTION
**Pull request by markopoloparadox**

### CODE REVIEW

File: avail-rust/src/full/transactions/options.rs [hunk 1]
```diff
@@ -1,4 +1,4 @@
-use crate::rpcs::Rpc;
+use super::super::rpcs::Rpc;
```
Comment (fmiguelgarcia) [inline avail-rust/src/full/transactions/options.rs:1]: too much `super` :D

File: avail-rust/src/full/transactions/session.rs [hunk 2]
```diff
@@ -1,7 +1,7 @@
+use super::super::rpcs::Rpc;
```
Comment (fmiguelgarcia) [inline avail-rust/src/full/transactions/session.rs:1]: ditto

File: avail-rust/src/full/transactions/staking.rs [hunk 3]
```diff
@@ -1,8 +1,8 @@
-use crate::api_dev::api::runtime_types::pallet_staking::ValidatorPrefs;
-use crate::api_dev::api::runtime_types::sp_arithmetic::per_things::Perbill;
-use crate::rpcs::Rpc;
-use crate::sdk::WaitFor;
+use super::super::rpcs::Rpc;
```
Comment (fmiguelgarcia) [inline avail-rust/src/full/transactions/staking.rs:1]: ditto

**Review by fmiguelgarcia — COMMENTED** [review]
Looks like you are working on a light version of `subxt`, I think we could move it out this repo, maybe `light-subxt`? :smile: 
So `avail-rust` could be a light ergonomic crate over `light-subxt` or `subxt`.
Another advantages is that a larger community could participate if the project is independent of avail node.
Migrate into a separate repo seems like a simplification on release procedure, so changes on `light-subxt` won't imply a new `avail-node` version.

**Comment by markopoloparadox** [comment]
> Looks like you are working on a light version of `subxt`, I think we could move it out this repo, maybe `light-subxt`? 😄 So `avail-rust` could be a light ergonomic crate over `light-subxt` or `subxt`. Another advantages is that a larger community could participate if the project is independent of avail node. Migrate into a separate repo seems like a simplification on release procedure, so changes on `light-subxt` won't imply a new `avail-node` version.

I agree, I will migrate it to a new repo

**Pull request closed without merging**